src/transaction/algo.ts

```diff
@@ -0,0 +1,54 @@
+const algosdk = require('algosdk');
+const base32 = require('base32.js');
+
+export const getClient = (testnet: boolean) => {
+    if (testnet) {
+        const baseServer = 'https://testnet-algorand.api.purestake.io/ps2'
+        const port = '';
+        const token = {'X-API-Key': 'klnvbVsn983Q3awiuWUGn2OKHkyi3U974JYtGvdt'}
+        const algodClient = new algosdk.Algodv2(token, baseServer, port);
+        return algodClient;
+    } else {
+        return null;
+    }
+}
+
+const waitForConfirmation = async (algodClient: any, txId: string) => {
+    let lastround = (await algodClient.status().do())['last-round'];
+    let limit = 0;
+    while (limit < 30) {
+        let pendingInfo = await algodClient.pendingTransactionInformation(txId).do();
+        if (pendingInfo['confirmed-round'] !== null && pendingInfo['confirmed-round'] > 0) {
+            return 1;
+            break;
+        }
+        lastround++;
+        limit++;
+        await algodClient.statusAfterBlock(lastround).do();
+    }
+    return 0;
+}
+
+export const signAlgoTransaction = async ( testnet: boolean, from: string, to: string, fee: number, amount: BigInt, privKey: string) => {
```

Comment:
missing documentation + parameter description

Reply:
missing unit test for that

Reply:
The unit test code is already written inside algo.spec.ts.
Should I move it to somewhere else?